libs/sdk-core/src/persist/channels.rs

```diff
@@ -108,6 +111,9 @@ impl SqliteStorage {
                     _ => c.closed_at,
                 },
                 c.funding_outnum,
+                c.closed_at,
```

Comment:
`closed_at` also set in the match statement above:
```
match c.state {
    ChannelState::Opened | ChannelState::PendingOpen => None,
    _ => c.closed_at,
},
```

Reply:
Fixed